test/models/history_test.rb

```diff
@@ -42,6 +42,22 @@ def history_types
     end
   end
 
+  test 'update patient updated_at upon history create, update, and delete' do
```

Comment:
Test name should be updated (we don't check update/delete for History).

Reply:
fixed